userland/lib/Makefile.in

```diff
@@ -95,8 +95,8 @@ OBJS      = pfring.o pfring_mod.o pfring_utils.o pfring_mod_stack.o pfring_hw_fi
 # CROSS_COMPILE=arm-mv5sft-linux-gnueabi-
 #
 CC        = ${CROSS_COMPILE}gcc
-CFLAGS    = @CFLAGS@ -Wall -fPIC ${INCLUDE} @HAVE_PF_RING_ZC@ @HAVE_PF_RING_FT@ @HAVE_AF_XDP@ @HAVE_DAG@ @HAVE_FIBERBLAZE@ @HAVE_ACCOLADE@ @HAVE_MYRICOM@ @HAVE_NETCOPE@ @HAVE_EXABLAZE@ @HAVE_BPF@ @HAVE_HW_TIMESTAMP@ @HAVE_NT@ @HAVE_NPCAP@
```

Comment:
Why did you remove @CFLAGS@? This is computed by the configure to add optiomization or debug flags